package/MDAnalysis/core/groups.py

```diff
@@ -155,6 +213,7 @@ def __getitem__(self, item):
             if isinstance(item, list) and item:  # check for empty list
                 # hack to make lists into numpy arrays
                 # important for boolean slicing
+                # TODO: what about tuples?
```

Comment:
WRT tuples, I've tried to make slicing a Group identical to slicing a numpy array.  Slicing a np array with a tuple seems to do some sort of weird n-dimensional slice, so `a[(1, 2)] == a[1][2]`.  Happy to change this over if it's a problem though